cpp/perspective/src/cpp/emscripten.cpp

```diff
@@ -1511,6 +1511,30 @@ namespace binding {
      *
      * Context API
      */
+    template <>
+    std::shared_ptr<t_ctxunit>
+    make_context(std::shared_ptr<Table> table, std::shared_ptr<t_schema> schema,
+        std::shared_ptr<t_view_config> view_config, const std::string& name) {
+        auto columns = view_config->get_columns();
+        auto filter_op = view_config->get_filter_op();
+        auto fterm = view_config->get_fterm();
+        auto computed_columns = view_config->get_computed_columns();
```

Comment:
Can we add a `t_config` which initializes these to the empty values we alreayd know these to be?